server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/filter/ContentMatcher.java

```diff
@@ -57,6 +61,26 @@ private AddressHeader(String fullAddress) {
                 return Optional.empty();
             }
         }
+
+        boolean exactMatch(AddressHeader other) {
```

Comment:
exacMatch is in general for exactCaseMatch. So you mean equalsIgnoreCase?

Reply:
`matchesIgnoreCase` is better as

`{"personal":"me", "address":null}` should match `{"personal":"me", "address":"me@domain.tld"}`